src/routes/issue-trigger-rule/getByClient.ts

```diff
@@ -0,0 +1,35 @@
+import Joi from 'joi';
```

Comment:
I think naming this file find and then using: 
  method: 'get',
  path: '/',
  query: etc
would be more consistent with the rest of the project